src/components/AddToDo.js

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from 'react';
+import { View, Text, TextInput } from 'react-native';
+import Button from './Button';
+import styles from './AddToDo.styles';
+import Header from './Header';
+import strings from '../assets/strings';
+
+class AddToDo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { taskTitle: '', taskDescription: '', error: '' };
+  }
+
+  validateNewTask = () => {
+    const { addToDo } = this.props;
+    const { emptyTextOrDescription } = strings;
+    if (this.state.taskTitle.length !== 0 && this.state.taskDescription.length !== 0) {
+      addToDo(this.state.taskTitle, this.state.taskDescription);
+    } else {
+      this.setState({ error: emptyTextOrDescription });
+    }
+  }
+
+  renderError = () => {
+    const { errorContainer, errorStyle } = styles;
+    if (this.state.error.length !== 0) {
+      return <View style={errorContainer}><Text style={errorStyle}>{this.state.error}</Text></View>;
+    }
+    return <View />;
+  }
+
+  render() {
+    const { onClickAction } = this.props;
+    const {
+      container, title, description, actions, borderBottom, borderText, rightActionContainer, leftActionContainer, centerContainer,
+    } = styles;
+    const { save, cancel, newTask } = strings;
+    return (
+      <Fragment>
+        <Header>
+          <View style={actions}>
+            <Button onClickAction={onClickAction}>
```

Comment:
I think it would be a good idea to be a little more descriptive with the function names